index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en"> 
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@100;200&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+     <img  class="alignleft" src="image_geometry_1.svg" alt="header image">
+     <img  class="alignright" src="ic_menu.svg" alt="ic-menu image">
```

Comment:
- [ ] Kindly wrap the mobile menu button in a `nav` tag for semantic reasons 👍

Reply:
Fixed.